api/src/test/java/com/graphhopper/util/GPXEntry_DiffblueTest.java

```diff
@@ -0,0 +1,73 @@
+/**
+ *
+ *
+ * Copyright 2018 Diffblue Limited
+ *
+ * Diffblue Limited licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.graphhopper.util;
+
+import com.graphhopper.util.GPXEntry;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class GPXEntry_DiffblueTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+
+  /* testedClasses: GPXEntry */
+
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 65 branch to line 68
+   *  - conditional line 68 branch to line 68
+   *  - conditional line 69 branch to line 69
+   */
+
+  @Test
+  public void equalsInputNotNullOutputFalse() {
+
+    // Arrange
+    final GPXEntry objectUnderTest = new GPXEntry(0.0, 0.0, 0.0, -6_935_684_335_438_659_590L);
+    final GPXEntry obj = new GPXEntry(0.0, 0.0, 0.0, -6_935_684_335_438_921_733L);
+
+    // Act
+    final boolean retval = objectUnderTest.equals(obj);
+
+    // Assert result
+    Assert.assertEquals(false, retval);
+  }
+
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 65 branch to line 66
+   */
+
+  @Test
+  public void equalsInputNullOutputFalse() {
+
+    // Arrange
+    final GPXEntry objectUnderTest = new GPXEntry(0.0, 0.0, 0.0, -1L);
+    final Object obj = null;
+
+    // Act
+    final boolean retval = objectUnderTest.equals(obj);
+
+    // Assert result
+    Assert.assertEquals(false, retval);
+  }
```

Comment:
Did the positive test case have too much reflection?  The rests are 👍

Reply:
or is it already covered?

Reply:
There was no positive test case generated.